### PR TITLE
better tagline

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

**Comment by gianm** [comment]
sure why not 👍

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <div class="container">
   <div class="row">
     <div class="col-md-8 col-md-offset-2 text-center">
-    <p class="lead"><b>Druid</b> is a fast column-oriented distributed data store.</p>
+    <p class="lead"><b>Druid</b> is a high-performance column-oriented distributed data store.</p>
```
Comment (drcrallen) [inline index.html]: I think `high-performance, column-oriented` is proper structure

Comment (drcrallen) [inline index.html]: but my `,` rules are pretty rusty

Comment (fjy) [inline index.html]: fixed

I wasn't sure either and was going back and forth on it

**Pull request merged**